aether-common-module/aether/common/utils.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2018 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from django.conf import settings
+
+
+def resolve_file_url(url):
+    '''
+    Make all file urls absolute.
+    '''
+    if url.startswith('/'):
+        # For local development, the environment variable DJANGO_REMOTE_STORAGE is
+        # set to "filesytem" and files (attachments, media files etc.) are stored on
+        # the filesystem and served via nginx. Example URL:
+        # http://odk.aether.local/media/<path-to-file>.
+        return f'http://{settings.HOSTNAME}{url}'
```

Comment:
What happens with `https`? This is the expected protocol in production servers :grimacing:

Reply:
In production, only the absolute url will be used and that one uses https (see comments).
Not great to hardcode the http scheme though, do you have any ideas?

Reply:
You can use the `SECURE_PROXY_SSL_HEADER` variable to detect the current protocol.

https://github.com/eHealthAfrica/aether/blob/develop/aether-common-module/aether/common/conf/settings.py#L244